### DISCUSSION TITLE
EC performance

### DISCUSSION THREAD

**Discussion by Lucas12138** [apache/ozone#6722] [FAQ]
Hi community,

We changed our replication type from Ratis One to EC RS-6-3-1024k.
We did some performance benchmarking and saw for downloading a 1TB file,
Ratis One took around 1500s and EC RS-6-3-1024k took around 4000s.
**We are wondering is this mush of the performance difference is expected?**

**Also, we're curious if XOR-2-1-1024k might have better or worse performance?**

Besides, I saw that "Enabling ISA-L allows significantly improve EC performance." For the performance here, **is it more about uploading speed, downloading speed or storage usage?**

Thanks!

**Comment by kerneltime** [comment]
@Lucas12138 if you used Ratis One, there is only a single copy created and that will be faster than reading multiple erasure coded shards. For diving deeper into why the erasure coded copy takes longer, I would recommend to start dashboard-ing  metrics and providing more details about the setup, commands used and configuration.
